test/base/test_utils.py

```diff
@@ -188,6 +188,19 @@ class Foo(object):
         eq_(len(w), 2)
         eq_(len(w._storage), 2)
 
+    @testing.requires.predictable_gc
```

Comment:
**mike bayer** (zzzeek) wrote:

TODO: move this to the test_memusage suite once merged and we rebase